app/Community/Controllers/MessageController.php

```diff
@@ -38,9 +38,20 @@ public function store(MessageRequest $request): RedirectResponse
                 return back()->withErrors(__('legacy.error.error'));
             }
 
+            foreach ($thread->users as $threadUser) {
+                if (!$threadUser->is($user) && !$user->can('sendToRecipient', [Message::class, $threadUser])) {
+                    return back()->withErrors(__('legacy.error.permissions'));
```

Comment:
This error is misleading. "Insufficient permissions". Something along the lines of "Unable to complete request" might work better.

Reply:
I went ahead and went with:
> This user isn't accepting messages right now.

This seems like the right balance of not giving away too much information, but also not being overly vague on what is happening. I'm afraid "unable to complete request" is inevitably going to lead to people filing bug reports.